### DISCUSSION TITLE
Put soft deletion on orders

### DISCUSSION THREAD

**Discussion by StephaneBour** [lunarphp/lunar#1927] [Feature Requests]
In my case, I can have orders canceled (either at my request or at the customer's request).

I find it interesting to be able to delete them and not see them in the statistics dashboards.

**Comment by glennjacobs** [comment]
We won't be introducing soft deletion of orders any time soon. A better solution to the problem might be to use a "Cancelled" order status and then introduce some config to the dashboard widgets to only consider certain statuses (or maybe exclude some).
